GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesList.cs

```diff
@@ -393,6 +445,37 @@ private static string ShortenText(string text, Font font, float maxWidth)
             return text;
         }
 
+        private bool PromptCategoryName(List<string> categories, string originalName, out string name)
+        {
+            using (var dialog = new FormDashboardCategoryTitle(categories, originalName))
+            {
+                if (dialog.ShowDialog(this) == DialogResult.OK)
+                {
+                    name = dialog.Category;
+                    return true;
+                }
+
+                name = null;
+                return false;
+            }
+        }
+
+        private bool PromptUserConfirm(string question, string caption)
+        {
+            return MessageBox.Show(this, question, caption, MessageBoxButtons.OKCancel, MessageBoxIcon.Exclamation) ==
```

Comment:
I think `MessageBoxIcon.Question` would be more appropriate here. Any particular reason you chose "Exclamation"?

Also I suggest making the "No" button the default to prevent accidental (or habitual) hit on "Enter" to confirm.

Reply:
ok